packages/components/paginator/src/paginator.ts

```diff
@@ -345,8 +346,6 @@ export class Paginator extends ScopedElementsMixin(LitElement) {
           <sl-icon name="caret-right-solid"></sl-icon>
         </sl-button>
       </nav>
-      <!-- We want this to be read every time the active page changes. -->
-      <div id="live" aria-live="polite" aria-atomic="true">${msg(str`Page ${this.page} of ${this.#pages}`)}</div>
```

Comment:
When we remove it, it's not announced that when we change the page-size in `sl-paginator-size` that the active page is being changed into `1` (it's going back to the first page). 
Also it will not work with the grid (will not be announced), when we change the filtering or sorting (it also changes the page to page `1` again).